proto/cosmos/auth/v1beta1/query.proto

```diff
@@ -26,6 +26,21 @@ service Query {
   rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
     option (google.api.http).get = "/cosmos/auth/v1beta1/params";
   }
+
+  // Bech32 queries bech32Prefix
+  rpc Bech32Prefix(Bech32PrefixRequest) returns (Bech32PrefixResponse) {
+    option (google.api.http).get = "/cosmos/auth/v1beta1/bech32";
+  }
+
+  // AddressString converts Account Address bytes to string
+  rpc AddressString(AddressStringRequest) returns (AddressStringResponse) {
```

Comment:
```suggestion
  rpc AddressBytesToString(AddressStringRequest) returns (AddressStringResponse) {
```

my suggestion, to make it more explicit, and to match the AddressCodec

Reply:
Sounds good to me. However, this suggestion introduces the name conflict with @aaronc `AddressCodec` methods [suggestion](https://github.com/cosmos/cosmos-sdk/pull/9759#discussion_r679328736)
```
type AddressCodec interface {
	// AddressStringToBytes encodes text to bytes
	AddressStringToBytes(text string) ([]byte, error)
	// AddressBytesToString encodes bytes to text
	AddressBytesToString(bz []byte) (string, error)
}
```

Reply:
My suggestion was actually to exactly match the rpc method with the AddressCodec method

Reply:
Matching name of rpc with `AddressCodec` methods causes name conflict since methods are put on the same `AccountKeeper` struct. To resolve this issue I suggest to name `AddressCodec` methods like that `convertAddressStringToBytes` and `convertAddressBytesToString`. Rpc methods remain the same AddressStringToBytes` and `AddressBytesToString` . Are you okay with that? cc @amaurym  @aaronc